### PR TITLE
Combine DetectAndApplyRecommendations examples.

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/Recommendations/DetectAndApplyRecommendations.php [hunk 1]
```diff
@@ -124,57 +121,109 @@ public static function runExample(GoogleAdsClient $googleAdsClient, int $custome
...
+        foreach ($response->getResults() as $appliedRecommendation) {
+            /** @var Recommendation $appliedRecommendation */
+            printf(
+                "Applied a recommendation with resource name: '%s'.%s",
+                $appliedRecommendation->getResourceName(),
+                PHP_EOL
+            );
         }
     }
+    // [EMD apply_recommendation]
```
Comment (Raibaz) [inline examples/Recommendations/DetectAndApplyRecommendations.php:226]: Typo! :)

**Review by Raibaz — CHANGES_REQUESTED** [review]

File: examples/Recommendations/DetectAndApplyRecommendations.php [hunk 2]
```diff
@@ -124,57 +121,109 @@ public static function runExample(GoogleAdsClient $googleAdsClient, int $custome
...
+                    "\tKeyword = '%s'%s\ttype = '%s'%s",
+                    $keyword->getText(),
+                    PHP_EOL,
+                    KeywordMatchType::name($keyword->getMatchType()),
                     PHP_EOL
                 );
-                sleep(self::PERIOD_IN_SECONDS);
             }
+            // Creates an ApplyRecommendationOperation that will be used to apply this
+            // recommendation, and add it to the list of operations.
```
Comment (laurachevalier4) [inline examples/Recommendations/DetectAndApplyRecommendations.php:157]: Nit: and "adds" it to the list (for consistency with "Creates")

File: examples/Recommendations/DetectAndApplyRecommendations.php [hunk 3]
```diff
@@ -124,57 +121,109 @@ public static function runExample(GoogleAdsClient $googleAdsClient, int $custome
...
+        }
+    }
+
+    /**
+     * Creates a ApplyRecommendationOperation to apply the given recommendation.
+     *
+     * @param string $recommendationResourceName the resource name of the recommendation to apply
+     * @return ApplyRecommendationOperation the created ApplyRecommendationOperation
+     */
+    private static function buildRecommendationOperation(
```
Comment (laurachevalier4) [inline examples/Recommendations/DetectAndApplyRecommendations.php:172]: Per the Perl changes, can you add `build_apply_recommendation_operation` tags around this example? See https://github.com/googleads/google-ads-perl/pull/124/files

File: examples/Recommendations/DetectAndApplyRecommendations.php [hunk 4]
```diff
@@ -124,57 +121,109 @@ public static function runExample(GoogleAdsClient $googleAdsClient, int $custome
     {
         $googleAdsServiceClient = $googleAdsClient->getGoogleAdsServiceClient();
         // Creates a query that retrieves keyword recommendations.
```
Comment (laurachevalier4) [inline examples/Recommendations/DetectAndApplyRecommendations.php:123]: Per the Perl changes, can you add a `detect_keyword_recommendations` tag here, and a closing tag at line 160?

Comment (fiboknacky) [reply examples/Recommendations/DetectAndApplyRecommendations.php:123]: Done.

**Review by Raibaz — APPROVED** [review]

**Review by laurachevalier4 — APPROVED** [review]

**Pull request merged**